### DISCUSSION TITLE
Fill Em Up (GENERAL) [R]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#32] [Card Balance]
We've already talked on Discord.

1) Max hand size must be 7 at the end of your turn.
2) Play during your turn.

My suggestions for "Fill Em Up" and "Hidden cache" :

Fill Em Up (GENERAL) [R]
Play during your turn. Draw five cards for 5DP.

Hidden Cache (GENERAL) [U]
ANNIHILATED
Play at any time. You may immediately draw enough cards to fill your hand up to 7 cards. Skip your next draw step .

**Comment by Sol-au-fein** [comment]
Spot on

**Reply by framerate** [reply]
I do think I want to bring back the "must discard end of turn" rule. Just need to make a UI for it.

Regarding the rest:

> Play During Your Turn
This changes the complete "feel" of the original card. I try not to do that because people always crucify me. You guys think that's ok, though? The 5DP thing seems ok but see below

So... we have started adding DP cost to a lot of cards that balance things. HOWEVER. We originally changed Positive Karma to give a lot less DP (generally I think we made it a little harder to get DP)

Do you think we need to make cards like Positive Karma give more again?

**Reply by Sol-au-fein** [reply]
Not at this point. 
Current DP sources are sufficient for now.

I vote for update Hidden Cache / Fill Em Up side effects now, because of Self Sacrifice card

Im not against Fill Em Up played anytime.

**Reply by 21-ric** [reply]
If we don't change "Fill em up" this card must be played during your turn or suspend.

If we add a DP cost we could play this card at any time in my opinion.

No need to change Positive Karma.

**Reply by Sol-au-fein** [reply]
Yep, so basically what Ric said in description with this change
Fill Em Up: 5DP, anytime
Hidden Cache: skip next draw, anytime

Adding Discarding down to 7 cards feature later

**Comment by Kite80** [comment]
Fill'em up as it is now must be balanced to avoid people play 2 or more of them and just skip one draw phase the following turn. It should be each time this card is played, player skips his next draw phase, whichever turn it would happen

**Reply by Sol-au-fein** [reply]
This side effect is completely useless for Fill Em Up and must be changed. I play Fill Em Up when i have full hand so i wouldnt draw anyway next turn.
Hidden Cache should skip next draw. Fill Em Up should be paid.

**Comment by TomekPL-1** [comment]
I think that problem with 'fill em up' can be solved if there were rule that on the end of your turn if you have more cards in your hand that your hand limit allowes you need to discard extra cards.

**Reply by Sol-au-fein** [reply]
Yes, definetely. But that require new code.

**Reply by framerate** [reply]
True, but it's actually on the road map. Cards like Tekron and cards that trigger combat are related and should lead to this feature being added for launch.

**Reply by Kite80** [reply]
I agree with this solution: a player could still draw half his deck in one turn, but then he has to play as many cards as possible before the end. It's still an advantage but it lasts for one turn, not for the entire game
